libs/core/src/lib/combobox/combobox.component.ts

```diff
@@ -420,10 +420,7 @@ export class ComboboxComponent implements ComboboxInterface, ControlValueAccesso
             event.stopPropagation();
         } else if (this.openOnKeyboardEvent && !event.ctrlKey && !KeyUtil.isKeyCode(event, this.nonOpeningKeys)) {
             this.isOpenChangeHandle(true);
-            const acceptedKeys = !KeyUtil.isKeyCode(event, BACKSPACE)
-                && !KeyUtil.isKeyType(event, 'alphabetical')
-                && !KeyUtil.isKeyType(event, 'numeric');
-            if (this.isEmptyValue && acceptedKeys) {
+            if (this.isEmptyValue && (KeyUtil.isKeyCode(event, SPACE) || KeyUtil.isKeyType(event, 'control'))) {
```

Comment:
```suggestion
            if (this.isEmptyValue && KeyUtil.isKeyCode(event, [SPACE, CONTROL])) {
```
please use CONTROL from @angular/cdk/keycodes